inc/config.php

```diff
@@ -930,7 +930,8 @@
 	$config['stylesheets']['Yotsuba B'] = ''; // Default; there is no additional/custom stylesheet for this.
 	$config['stylesheets']['Yotsuba'] = 'yotsuba.css';
 	// $config['stylesheets']['Futaba'] = 'futaba.css';
-	// $config['stylesheets']['Dark'] = 'dark.css';
+	$config['stylesheets']['Dark'] = 'dark.css';
```

Comment:
Let's keep to editing instance-config.php rather than this file

Reply:
changed the theme changes to instance config and reverted changes to config.php